File: app/src/main/java/com/casebeaumonde/fragments/profile/profileResponse/EditProfileResponse.java
package com.casebeaumonde.fragments.profile.profileResponse;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class EditProfileResponse {

@SerializedName("code")
@Expose
private String code;
@SerializedName("data")
@Expose
private Data data;

public String getCode() {
return code;
}

public void setCode(String code) {
this.code = code;
}

public Data getData() {
return data;
}

public void setData(Data data) {
this.data = data;
}

    public class Data {

        @SerializedName("user")
        @Expose
        private User user;

        public User getUser() {
            return user;
        }

        public void setUser(User user) {
            this.user = user;
        }

    }

    public class User {

        @SerializedName("id")
        @Expose
        private Integer id;
        @SerializedName("firstname")
        @Expose
        private String firstname;
        @SerializedName("lastname")
        @Expose
        private String lastname;
        @SerializedName("fullname")
        @Expose
        private String fullname;
        @SerializedName("phone")
        @Expose
        private String phone;
        @SerializedName("email")
        @Expose
        private String email;
        @SerializedName("chat_invitation")
        @Expose
        private String chat_invitation;

        public String getChat_invitation() {
            return chat_invitation;
        }

        public void setChat_invitation(String chat_invitation) {
            this.chat_invitation = chat_invitation;
        }

        @SerializedName("company")
        @Expose
        private String company;
        @SerializedName("status")
        @Expose
        private String status;
        @SerializedName("affiliate_id")
        @Expose
        private Integer affiliateId;
        @SerializedName("percent")
        @Expose
        private String percent;
        @SerializedName("paypal_email")
        @Expose
        private String paypal_email;

        public String getPaypal_email() {
            return paypal_email;
        }

        public void setPaypal_email(String paypal_email) {
            this.paypal_email = paypal_email;
        }

        @SerializedName("pending_balance")
        @Expose
        private String pendingBalance;
        @SerializedName("available_balance")
        @Expose
        private String availableBalance;
        @SerializedName("allow_hire")
        @Expose
        private Integer allowHire;
        @SerializedName("avatar")
        @Expose
        private String avatar;
        @SerializedName("email_verified_at")
        @Expose
        private String emailVerifiedAt;
        @SerializedName("created_at")
        @Expose
        private String createdAt;
        @SerializedName("updated_at")
        @Expose
        private String updatedAt;
        @SerializedName("deleted_at")
        @Expose
        private Object deletedAt;
        @SerializedName("role")
        @Expose
        private String role;
        @SerializedName("unread_notifications")
        @Expose
        private List<Object> unreadNotifications = null;
        @SerializedName("profile")
        @Expose
        private Profile profile;

        public Integer getId() {
            return id;
        }

        public void setId(Integer id) {
            this.id = id;
        }

        public String getFirstname() {
            return firstname;
        }

        public void setFirstname(String firstname) {
            this.firstname = firstname;
        }

        public String getLastname() {
            return lastname;
        }

        public void setLastname(String lastname) {
            this.lastname = lastname;
        }

        public String getFullname() {
            return fullname;
        }

        public void setFullname(String fullname) {
            this.fullname = fullname;
        }

        public String getPhone() {
            return phone;
        }

        public void setPhone(String phone) {
            this.phone = phone;
        }

        public String getEmail() {
            return email;
        }

        public void setEmail(String email) {
            this.email = email;
        }

        public String getCompany() {
            return company;
        }

        public void setCompany(String company) {
            this.company = company;
        }

        public String getStatus() {
            return status;
        }

        public void setStatus(String status) {
            this.status = status;
        }

        public Integer getAffiliateId() {
            return affiliateId;
        }

        public void setAffiliateId(Integer affiliateId) {
            this.affiliateId = affiliateId;
        }

        public String getPercent() {
            return percent;
        }

        public void setPercent(String percent) {
            this.percent = percent;
        }

        public String getPendingBalance() {
            return pendingBalance;
        }

        public void setPendingBalance(String pendingBalance) {
            this.pendingBalance = pendingBalance;
        }

        public String getAvailableBalance() {
            return availableBalance;
        }

        public void setAvailableBalance(String availableBalance) {
            this.availableBalance = availableBalance;
        }

        public Integer getAllowHire() {
            return allowHire;
        }

        public void setAllowHire(Integer allowHire) {
            this.allowHire = allowHire;
        }

        public String getAvatar() {
            return avatar;
        }

        public void setAvatar(String avatar) {
            this.avatar = avatar;
        }

        public String getEmailVerifiedAt() {
            return emailVerifiedAt;
        }

        public void setEmailVerifiedAt(String emailVerifiedAt) {
            this.emailVerifiedAt = emailVerifiedAt;
        }

        public String getCreatedAt() {
            return createdAt;
        }

        public void setCreatedAt(String createdAt) {
            this.createdAt = createdAt;
        }

        public String getUpdatedAt() {
            return updatedAt;
        }

        public void setUpdatedAt(String updatedAt) {
            this.updatedAt = updatedAt;
        }

        public Object getDeletedAt() {
            return deletedAt;
        }

        public void setDeletedAt(Object deletedAt) {
            this.deletedAt = deletedAt;
        }

        public String getRole() {
            return role;
        }

        public void setRole(String role) {
            this.role = role;
        }

        public List<Object> getUnreadNotifications() {
            return unreadNotifications;
        }

        public void setUnreadNotifications(List<Object> unreadNotifications) {
            this.unreadNotifications = unreadNotifications;
        }

        public Profile getProfile() {
            return profile;
        }

        public void setProfile(Profile profile) {
            this.profile = profile;
        }

    }

    public class Profile {

        @SerializedName("id")
        @Expose
        private Integer id;
        @SerializedName("user_id")
        @Expose
        private Integer userId;
        @SerializedName("about_me")
        @Expose
        private String aboutMe;
        @SerializedName("created_at")
        @Expose
        private String createdAt;
        @SerializedName("updated_at")
        @Expose
        private String updatedAt;
        @SerializedName("deleted_at")
        @Expose
        private Object deletedAt;

        public Integer getId() {
            return id;
        }

        public void setId(Integer id) {
            this.id = id;
        }

        public Integer getUserId() {
            return userId;
        }

        public void setUserId(Integer userId) {
            this.userId = userId;
        }

        public String getAboutMe() {
            return aboutMe;
        }

        public void setAboutMe(String aboutMe) {
            this.aboutMe = aboutMe;
        }

        public String getCreatedAt() {
            return createdAt;
        }

        public void setCreatedAt(String createdAt) {
            this.createdAt = createdAt;
        }

        public String getUpdatedAt() {
            return updatedAt;
        }

        public void setUpdatedAt(String updatedAt) {
            this.updatedAt = updatedAt;
        }

        public Object getDeletedAt() {
            return deletedAt;
        }

        public void setDeletedAt(Object deletedAt) {
            this.deletedAt = deletedAt;
        }

    }
}

File: app/src/main/java/com/casebeaumonde/fragments/profile/profileResponse/EditProfileResponse.kt
package com.casebeaumonde.fragments.profile.profileResponse

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class EditProfileResponse {

    @SerializedName("code")
    @Expose
    private var code: String? = null

    @SerializedName("data")
    @Expose
    private var data: Data? = null

    fun getCode(): String? {
        return code
    }

    fun setCode(code: String?) {
        this.code = code
    }

    fun getData(): Data? {
        return data
    }

    fun setData(data: Data?) {
        this.data = data
    }

    class Data {
        @SerializedName("user")
        @Expose
        var user: User? = null
    }

    class User {
        @SerializedName("id")
        @Expose
        var id: Int? = null

        @SerializedName("firstname")
        @Expose
        var firstname: String? = null

        @SerializedName("lastname")
        @Expose
        var lastname: String? = null

        @SerializedName("fullname")
        @Expose
        var fullname: String? = null

        @SerializedName("phone")
        @Expose
        var phone: String? = null

        @SerializedName("email")
        @Expose
        var email: String? = null

        @SerializedName("chat_invitation")
        @Expose
        var chat_invitation: String? = null

        @SerializedName("company")
        @Expose
        var company: String? = null

        @SerializedName("status")
        @Expose
        var status: String? = null

        @SerializedName("affiliate_id")
        @Expose
        var affiliateId: Int? = null

        @SerializedName("percent")
        @Expose
        var percent: String? = null

        @SerializedName("paypal_email")
        @Expose
        var paypal_email: String? = null

        @SerializedName("pending_balance")
        @Expose
        var pendingBalance: String? = null

        @SerializedName("available_balance")
        @Expose
        var availableBalance: String? = null

        @SerializedName("allow_hire")
        @Expose
        var allowHire: Int? = null

        @SerializedName("avatar")
        @Expose
        var avatar: String? = null

        @SerializedName("email_verified_at")
        @Expose
        var emailVerifiedAt: String? = null

        @SerializedName("created_at")
        @Expose
        var createdAt: String? = null

        @SerializedName("updated_at")
        @Expose
        var updatedAt: String? = null

        @SerializedName("deleted_at")
        @Expose
        var deletedAt: Any? = null

        @SerializedName("role")
        @Expose
        var role: String? = null

        @SerializedName("unread_notifications")
        @Expose
        var unreadNotifications: List<Any>? = null

        @SerializedName("profile")
        @Expose
        var profile: Profile? = null
    }

    class Profile {
        @SerializedName("id")
        @Expose
        var id: Int? = null

        @SerializedName("user_id")
        @Expose
        var userId: Int? = null

        @SerializedName("about_me")
        @Expose
        var aboutMe: String? = null

        @SerializedName("created_at")
        @Expose
        var createdAt: String? = null

        @SerializedName("updated_at")
        @Expose
        var updatedAt: String? = null

        @SerializedName("deleted_at")
        @Expose
        var deletedAt: Any? = null
    }
}